test/Dapr.Client.Test/DaprClientTest.cs

```diff
@@ -29,11 +29,7 @@ public void CreateInvokeHttpClient_WithAppId()
         [Fact]
         public void CreateInvokeHttpClient_InvalidAppId()
         {
-            var ex = Assert.Throws<ArgumentException>(() => 
-            { 
-                // The appId needs to be something that can be used as hostname in a URI.
```

Comment:
I think it's useful to retain this comment in the test as it provides more context to the test name and what it's covering